.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
Q: that means that only templates can be used? Isn't one of the templates a "blanks issue" template? I guess the difference is in auto adding the `needs-triage` label?

Reply:
exactly 👍